### PR TITLE
create seq2mezz.py. recursive seq to proresHQ

### PR DESCRIPTION
**Pull request by raecasey**
modified 3 lines from seq2dv script

### CODE REVIEW

File: seq2mezz.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+ 
+
+def make_dv(ffmpeg_friendly_name, output):
+    ffmpeg_friendly_name += '%06d.tiff'
+    cmd = ['ffmpeg','-f','image2',
+           '-framerate','24',y
+           '-i', ffmpeg_friendly_name,
+           '-c:v','prores',
+           '-profile', '3',
+           '-vf','copy',
```
Comment (kieranjol) [inline seq2mezz.py]: This line should be removed. `-vf` stands for 'video filter'. It's needed I'm seq2dv to rescale to an dv friendly resolution like 720/576. The `copy` argument here would probably result on an ffmpeg error. I'm assuming that you don't want rescaling here?

File: seq2mezz.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    return ffmpeg_friendly_name, output
+ 
+
+def make_dv(ffmpeg_friendly_name, output):
+    ffmpeg_friendly_name += '%06d.tiff'
+    cmd = ['ffmpeg','-f','image2',
+           '-framerate','24',y
+           '-i', ffmpeg_friendly_name,
+           '-c:v','prores',
+           '-profile', '3',
```
Comment (kieranjol) [inline seq2mezz.py]: Off the top of my head, I think you need to use `-profile:v` here instead.

File: seq2mezz.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        counter += 1
+    dirname = os.path.dirname(source_directory)
+    output = dirname + '/%s_consolidate.mov' % os.path.split(source_directory)[-1]
+    return ffmpeg_friendly_name, output
+ 
+
+def make_dv(ffmpeg_friendly_name, output):
+    ffmpeg_friendly_name += '%06d.tiff'
+    cmd = ['ffmpeg','-f','image2',
+           '-framerate','24',y
```
Comment (kieranjol) [inline seq2mezz.py]: The `y` needs to be removed from the end of this line

**Comment by kieranjol** [comment]
Cheers Raelene, I left some comments on specific lines. It might make sense to add an argparse prores argument to seq2dv instead to reduce the code repetition and the need for maintenance, but this might be useful short term.

File: seq2mezz.py (hunk 1 above)
Comment (raecasey) [inline seq2mezz.py]: yes, that's correct. no rescaling wanted. the copy argument didn't disrupt things though. the script works and outputs the source raster in pro resHQ

**Comment by kieranjol** [comment]
LGTM, thx!

**Pull request merged**